components/dropdown/src/floatingUI.mjs

```diff
@@ -265,8 +307,11 @@ export default class AuroFloatingUI {
   configure(elem) {
     this.element = elem;
     this.element.trigger = this.element.shadowRoot.querySelector('#trigger');
-    this.element.bib = this.element.shadowRoot.querySelector('#bib');
     this.element.triggerChevron = this.element.shadowRoot.querySelector('#showStateIcon');
+    this.element.bib = this.element.shadowRoot.querySelector('#bib');
+    this.element.bibSizer = this.element.shadowRoot.querySelector('#bibSizer');
+    
+    document.body.append(this.element.bib);
```

Comment:
I am confused why this was added. Help me understand?

Reply:
this is the part of the floatingUI's logic which takes #bib out of the dropdown and teleport to the body

Reply:
this line is moved from `auro-dropdown.js` to consolidate all floating logic to be in `floatingUI.mjs`